ckanext/qa/i18n/en_AU/LC_MESSAGES/ckanext-qa.po

```diff
@@ -0,0 +1,250 @@
+msgid "average_stars"
+msgstr "average_ticks"
+
+#: ckanext/qa/reports.py:165
+msgid "openness"
+msgstr "data_usability_rating"
+
+#: ckanext/qa/reports.py:166
+msgid "Openness (Five Stars)"
+msgstr "Data usability rating"
+
+#: ckanext/qa/reports.py:167
+msgid ""
+"Datasets graded on Tim Berners Lees' Five Stars of Openness - openly "
+"licensed, openly accessible, structured, open format, URIs for entities, "
+"linked."
+msgstr ""
+"Datasets rated according to rules similar to those used by the "
+"Australian Government's Linked Data Rating derived from "
+"(but different to) Tim Berners-Lee's Five Stars of Openness."
+
+#: ckanext/qa/tasks.py:251
+msgid "Could not understand the file format, therefore score is 1."
+msgstr ""
+
+#: ckanext/qa/tasks.py:261
+#, python-format
+msgid "Unknown error: %s"
+msgstr ""
+
+#: ckanext/qa/tasks.py:275
+msgid "License not open"
+msgstr ""
+
+#: ckanext/qa/tasks.py:302
+msgid "File could not be downloaded."
+msgstr ""
+
+#: ckanext/qa/tasks.py:303 ckanext/qa/tasks.py:378 ckanext/qa/tasks.py:383
+#: ckanext/qa/templates/qa/openness_stars.html:24
+#: ckanext/qa/templates/qa/openness_stars_brief.html:23
+#: ckanext/qa/templates/report/openness.html:88
+msgid "Reason"
+msgstr ""
+
+#: ckanext/qa/tasks.py:304
+#, python-format
+msgid "Error details: %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:305
+#, python-format
+msgid "Attempted on %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:309 ckanext/qa/tasks.py:317
+#, python-format
+msgid "This URL last worked on: %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:311
+msgid "This was the first attempt."
+msgstr ""
+
+#: ckanext/qa/tasks.py:313
+#, python-format
+msgid "Tried %s times since %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:319
+msgid "This URL has not worked in the history of this tool."
+msgstr ""
+
+#: ckanext/qa/tasks.py:355 ckanext/qa/tasks.py:386
+msgid "This file had not been downloaded at the time of scoring it."
+msgstr ""
+
+#: ckanext/qa/tasks.py:360
+#, python-format
+msgid "Cache filepath does not exist: \"%s\"."
+msgstr ""
+
+#: ckanext/qa/tasks.py:368
+#, python-format
+msgid ""
+"Content of file appeared to be format \"%s\" which receives openness "
+"score: %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:372
+msgid "The format of the file was not recognized from its contents."
+msgstr ""
+
+#: ckanext/qa/tasks.py:377
+msgid "File was not downloaded deliberately"
+msgstr ""
+
+#: ckanext/qa/tasks.py:378 ckanext/qa/tasks.py:383
+msgid "Using other methods to determine file openness."
+msgstr ""
+
+#: ckanext/qa/tasks.py:382
+msgid "A system error occurred during downloading this file"
+msgstr ""
+
+#: ckanext/qa/tasks.py:403
+msgid "Could not determine a file extension in the URL."
+msgstr ""
+
+#: ckanext/qa/tasks.py:410
+#, python-format
+msgid "URL extension \"%s\" relates to format \"%s\" and receives score: %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:414
+#, python-format
+msgid ""
+"URL extension \"%s\" relates to format \"%s\" but a score for that format"
```

Comment:
This looks like a message string not an ID.

Reply:
@ThrawnCA Read above comment, msgid are strings marked for internationalisation.